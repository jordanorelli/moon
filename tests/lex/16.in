1nbz32k: doauhef
