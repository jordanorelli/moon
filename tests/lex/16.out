{t_name 1nbz32k}
{t_object_separator :}
{t_string doauhef}
